cmake/projects/xcb/hunter.cmake

```diff
@@ -35,7 +46,7 @@ hunter_add_version(
 )
 
 set(_xcb_text_files "")
-foreach(_x composite damage dpms dri2 dri3 glx present randr record render res screensaver shape shm sync xevie xf86dri xfixes xinerama xkb xprint xtest xv xvmc)
```

Comment:
This configuration should work for both 1.11 and 1.13, hence list should be formed based on analysis of `HUNTER_xcb_VERSION` variable.